Analytics Spec

SamDelaney opened the pull request.

euonymos commented on docs/analytics_spec.md:
````diff
@@ -0,0 +1,183 @@
+```typescript
+{
+  type: "AuctionViewed",
+  unitId: unit,
````
Is that `unitId` sort of event identifier?

euonymos commented on docs/analytics_spec.md:
````diff
@@ -0,0 +1,183 @@
+```
+
+### Auctions Announced
+Triggers when the `announceAuction` transaction succeeds.
````
Do we have an auction identifier here somewhere?

euonymos reviewed (changes requested):
LGTM @SamDelaney, dropped a couple of question.

I think we have enough events to begin with, probably it's time to start moving from the other end, I mean reports and embedded widgets we are going to implement. From what comes to my mind:
* Delegate group performance (# auctions completed, # of auctions aborted)
* Seller statistics (#auctions completed)
* Bidder statistics (# bids, # wins, #failures to purchase)
* Lot summary (# of auctions, lowest and highest prices?)

SamDelaney commented:
@euonymos addressed your comments. Further development of the spec will require events & a demo project for testing, so I plan on closing this PR and revisiting later, barring further feedback.
